build/Rulesets/Roslyn.ruleset

```diff
@@ -1,8 +1,115 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<RuleSet Name="Common diagnostic rules for all Roslyn projects"
-         Description="This file contains diagnostic settings used by all Roslyn projects. Projects that need specific settings should have their own rule set files that Include this one, and then make the necessary adjustments." 
-         ToolsVersion="14.0">
-  <Rules AnalyzerId="Roslyn.Core" RuleNamespace="Roslyn.Core">
-    <Rule Id="AD0001" Action="Error" />
-  </Rules>
-</RuleSet>
+﻿<?xml version="1.0" encoding="utf-8"?> 
+<RuleSet Name="Common diagnostic rules to run during build for all shipping Roslyn projects" Description="This file contains diagnostic settings used by all Roslyn projects. Projects that need specific settings should have their own rule set files that Include this one, and then make the neceessary adjustments." ToolsVersion="14.0"> 
+  <IncludeAll Action="Hidden" /> 
+  <Rules AnalyzerId="Microsoft.ApiDesignGuidelines.Analyzers" RuleNamespace="Microsoft.ApiDesignGuidelines.Analyzers"> 
+    <Rule Id="CA1000" Action="None" /> 
```

Comment:
We should consider enabling lot more rules for this repo, while it is in its initial phase. Did you disable them as there were many violations/false positives OR was it because it was cloned from Roslyn repo?